src/app/views/search-route/search-route.component.css

```diff
@@ -71,6 +71,84 @@
   background-color: rgba(0,0,0,.2);
 }
 
+.search-route__route-information {
+  display: flex;
+  align-items: center;
+  background-color: var(--secondary);
+  color: white;
+  box-shadow: 0 2px 12px 0 rgba(0,0,0,.2);
+  padding: 1rem;
+  width: 100%;
+  z-index: 100;
+}
+
+.search-route__locations {
+  flex-grow: 1;
+  margin: 0 .5rem;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.search-route__nowrap-hidden {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.search-route__location-secondary {
+  opacity: .7;
+  font-size: .8rem;
+}
+
+.search-route__route-visualisation {
+  position: relative;
+}
+
+.search-route__route-visualisation-line {
+  left: 0.45rem;
+  top: 1.2rem;
+  height: .5rem;
```

Comment:
if you don't mind, please add a leading zero in front of the dot for consistency

also applies to numbers above and below